packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -446,12 +446,24 @@ const Docs: () => JSX.Element = () => {
         <SingleLineBetaBanner />
         <Description>
           The following examples show how to enable image insert functionality for message editing with rich text
-          editor. Under the `richTextEditorOptions` prop, an `onInsertInlineImage` callback to handle an inline image
-          that is inserted into the MessageThread component. This callback can be used to implement custom logic, such
-          as uploading the image to a server. After processing each inserted image in the callback, the results should
-          be passed back to the component through the `messagesInlineImagesWithProgress` prop for each message that has
-          inserted inline images. This prop will be used to render inline images in the MessageThread and submit them
-          with the message.
+          editor. Under the `richTextEditorOptions` prop, the `onInsertInlineImage` callback is used to handle each
```

Comment:
Can you make sure this is added to storybook8 as well since i think the MessageThread page has been migrated already